### ISSUE TITLE
Package creating multiple test runs when using cucumber

### ISSUE THREAD

**Issue by liamharries** [webdriverio-community/wdio-testrail-reporter#370]
I have the `oneReport` setting set to true however this still results in one test run created per feature file.

Am I misconfigured or is this a bug?

```
import { config as sharedConfig } from './wdio.conf.ts';

export const config: WebdriverIO.Config = {
    ...sharedConfig,     
    capabilities: [
        {
          browserName: "chrome",
          'goog:chromeOptions': {
            args: ['headless', 'disable-gpu']
        }
        },
      ],
      reporters:[
        "spec",
        // "json", // uncomment to place json in the console output for entire run
        "cucumberjs-json",
        [
          'testrail',
          {
            projectId: 36,
            suiteId: 1, // invalid suite generates a run
            domain: 'company.testrail.io',
            username: 'liam.harries@company.com',
            apiToken: 'TOKEN', 
            runName: `foo-web-qa ${new Date()}`,
            oneReport: true,
            includeAll: true,
            useCucumber: true,
            caseIdTagPrefix: 'C', // used only for multi-platform Cucumber Scenarios
          },
        ],
      ]
}
```

**Comment by christian-bromann** [comment]
Thanks for reporting, any contributions would be appreciated.

**Comment by MaikelBruin** [comment]
@liamharries isn't your problem a potential duplicate of [this issue](https://github.com/webdriverio-community/wdio-testrail-reporter/issues/397)?
you use `foo-web-qa ${new Date()}` as your run name which is dynamic. if you have multiple workers executing your tests, this name will be different for each worker, hence making different runs

**Comment by MaikelBruin** [comment]
@liamharries the latest version 0.4.0 adds an option to use an existing runId to update instead of creating a new run.
https://github.com/webdriverio-community/wdio-testrail-reporter/pull/490
personally I create a new run in the onPrepare hook (which gets executed once before all workers start) and then use that runId in the testrail reporter options. this results in only a single run and report being created whilst also using different feature files.

**Comment by christian-bromann** [comment]
@MaikelBruin do you think we can close the issue?

**Comment by MaikelBruin** [comment]
@christian-bromann yes I think so given that fact that there is a fix/workaround and given the inactiveness of the author

**Comment by therealbrad** [comment]
@MaikelBruin My company moved away from TestRail shortly after getting this reporter working so I don't have any way to make changes or test things out, unfortunately. I'm hoping someone else can take it over.

**Comment by christian-bromann** [comment]
Thanks folks for the feedback 👍
